src/app/game/tic-tac-toe/page.tsx

```
@@ -0,0 +1,42 @@
+            
+            <div className={styles.selectionContainer}>
+                <div className={styles.gameHeader}>
+                    <div className={styles.gameIcon}>
+                        <Image src={TicTacToeImage} alt="Tic Tac Toe" width={80} height={70} />
+                    </div>
+                    <h1>Tic Tac Toe</h1>
+                    <h2 className={styles.modeHeading}>Choose your game mode</h2>
+                </div>
+
+                <div className={styles.modeContainer}>
+                    <Link href="/game/tic-tac-toe/single-player" className={styles.modeCard} aria-label="Play Tic Tac Toe vs AI" role="button" tabIndex={0}>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)